doc/dev_guide/writing_docs.rst

```diff
@@ -73,17 +73,22 @@ the documentation by running the ``make`` command in the ``hyperspy/doc``
 directory. For example ``make html`` will build the whole documentation in
 html format. See the ``make`` command documentation for more details.
 
-To install the documentation dependencies, run either
+To install the documentation dependencies, you can use either
+``pip`` or ``conda`` from the root folder of the repository:
 
-.. code-block:: bash    
+.. tab-set::
```

Comment:
Nice layout with the tabs, but then we should maybe use the same for the installation of the test dependencies?